org.opent2t.sample.thermostat.superpopular/com.smartthings.thermostat/js/thingTranslator.js

```diff
@@ -285,20 +462,28 @@ class Translator {
      * Updates the specified resource with the provided payload.
      */
     postDeviceResource(di, resourceId, payload) {
-        if (di === generateGUID( this.controlId + 'opent2t.d.thermostat') )
-        {
-            var putPayload = resourceSchemaToProviderSchema(resourceId, payload);
-
-            return this.smartThingsHub.putDeviceDetailsAsync(this.endpointUri, this.controlId, putPayload)
-                .then((response) => {
-                    var schema = providerSchemaToPlatformSchema(response, true);
-                    return findResource(schema, di, resourceId);
-                });
+        if (di === generateGUID(this.controlId + 'opent2t.d.thermostat')) {
```

Comment:
suggestion: invert the condition to reduce nesting

Reply:
Adding to code clean up task.